packages/block-editor/src/components/global-styles/dimensions-panel.js

```diff
@@ -0,0 +1,520 @@
+	inheritedValue = value,
+	settings,
+	panelId,
+	defaultControls = DEFAULT_CONTROLS,
+} ) {
+	const decodeValue = ( rawValue ) =>
+		getValueFromVariable( { settings }, '', rawValue );
+
+	const showSpacingPresetsControl = useHasSpacingPresets( settings );
+	const units = useCustomUnits( {
+		availableUnits: settings?.spacing?.units || [
+			'%',
+			'px',
+			'em',
+			'rem',
+			'vw',
+		],
+	} );
+
+	// Content Size
+	const showContentSizeControl = useHasContentSize( settings );
+	const contentSizeValue = decodeValue( inheritedValue?.layout?.contentSize );
+	const setContentSizeValue = ( newValue ) => {
+		onChange( {
+			...value,
+			layout: {
+				...value?.layout,
+				contentSize: newValue,
+			},
+		} );
+	};
+	const hasUserSetContentSizeValue = () => !! value?.layout?.contentSize;
+	const resetContentSizeValue = () => setContentSizeValue( undefined );
+
+	// Wide Size
+	const showWideSizeControl = useHasWideSize( settings );
+	const wideSizeValue = decodeValue( inheritedValue?.layout?.wideSize );
+	const setWideSizeValue = ( newValue ) => {
+		onChange( {
+			...value,
+			layout: {
+				...value?.layout,
+				wideSize: newValue,
+			},
+		} );
+	};
+	const hasUserSetWideSizeValue = () => !! value?.layout?.wideSize;
+	const resetWideSizeValue = () => setWideSizeValue( undefined );
+
+	// Padding
+	const showPaddingControl = useHasPadding( settings );
+	const rawPadding = decodeValue( inheritedValue?.spacing?.padding );
+	const paddingValues = splitStyleValue( rawPadding );
+	const paddingSides = Array.isArray( settings?.spacing?.padding )
+		? settings?.spacing?.padding
+		: settings?.spacing?.padding?.sides;
+	const isAxialPadding =
+		paddingSides &&
+		paddingSides.some( ( side ) => AXIAL_SIDES.includes( side ) );
+	const setPaddingValues = ( newPaddingValues ) => {
+		const padding = filterValuesBySides( newPaddingValues, paddingSides );
+		onChange( {
+			...value,
+			spacing: {
+				...value?.spacing,
+				padding,
+			},
+		} );
+	};
+	const hasPaddingValue = () =>
+		!! value?.spacing?.padding &&
+		Object.keys( value?.spacing?.padding ).length;
+	const resetPaddingValue = () => setPaddingValues( undefined );
+
+	// Margin
+	const showMarginControl = useHasMargin( settings );
+	const rawMargin = decodeValue( inheritedValue?.spacing?.margin );
+	const marginValues = splitStyleValue( rawMargin );
+	const marginSides = Array.isArray( settings?.spacing?.margin )
+		? settings?.spacing?.margin
+		: settings?.spacing?.margin?.sides;
+	const isAxialMargin =
+		marginSides &&
+		marginSides.some( ( side ) => AXIAL_SIDES.includes( side ) );
+	const setMarginValues = ( newMarginValues ) => {
+		const margin = filterValuesBySides( newMarginValues, marginSides );
+		onChange( {
+			...value,
+			spacing: {
+				...value?.spacing,
+				margin,
+			},
+		} );
+	};
+	const hasMarginValue = () =>
+		!! value?.spacing?.margin &&
+		Object.keys( value?.spacing?.margin ).length;
+	const resetMarginValue = () => setPaddingValues( undefined );
```

Comment:
```suggestion
	const resetMarginValue = () => setMarginValues( undefined );
```

Small copy and paste typo?